src/components/card/card.tsx

```diff
@@ -132,6 +132,24 @@ const Card: React.FunctionComponent<CardProps> = ({
                     {secondaryTagElement}
                     <TypographyScale
                         variant={variant === 'large' ? 'heading5' : 'heading6'}
+                        sx={{
+                            ...(variant === 'large' && {
+                                fontSize: [
+                                    null,
+                                    'inc30',
+                                    'inc50',
+                                    'inc80',
+                                    null,
+                                ],
+                                lineHeight: [
+                                    null,
+                                    'inc20',
+                                    'inc30',
+                                    'inc50',
+                                    null,
+                                ],
```

Comment:
thanks for the feedback! No idea why these array of viewport sizes is taking me so to wrap my head around lol, ill try this out

Reply:
lol all good, it is kinda a weird concept at first so no worries!

Reply:
@mcolella14 I had to keep the second `inc30` because the font was going back to the larger size while still in the row layout, but this should be okay now

Reply:
Ah ok makes sense since you're overriding it, looks good to me!